src/convertToObject.js

```diff
@@ -6,7 +6,30 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+
+  // Крок 1: Очистка вхідного рядка
+  const cleanedString = sourceString
+    .replace(/[\n\r]/g, '') // Видаляємо символи нового рядка
+    .replace(/\s*;\s*/g, ';') // Видаляємо зайві пробіли навколо крапок з комою
+    .replace(/;\s*$/, ''); // Видаляємо останню крапку з комою
```

Comment:
All chained methods called on the same level should have the same indentation. This makes code much more readable. Align the method calls with the same indentation.